code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1278,3 +1255,77 @@
 
 	log_combat(victim, owner, "was drained.")
 	log_game("[key_name(victim)] was drained at [AREACOORD(victim.loc)].")
+
+/////////////////////////////////
+// Devour 
+/////////////////////////////////
+/datum/action/xeno_action/activable/devour
+	name = "Devour"
+	action_icon_state = "regurgitate"
+	mechanics_text = "Devour your victim to cocoon it in your belly. This cocoon will be automaticly ejected later, and until the marine inside it sill has life force it will give psychic point"
+	use_state_flags = XACT_USE_STAGGERED|XACT_USE_FORTIFIED|XACT_USE_CRESTED //can't use while staggered, defender fortified or crest down
```

Comment:
Grammar

Reply:
```suggestion
	mechanics_text = "Devour your victim to cocoon it in your belly. This cocoon will automatically be ejected later, and while the marine inside it still has life force it will give psychic points."
	use_state_flags = XACT_USE_STAGGERED|XACT_USE_FORTIFIED|XACT_USE_CRESTED //can't use while staggered, defender fortified or crest down
```